bldr/data/hooks/pre-init

```diff
@@ -1 +1,26 @@
 #!/bin/bash
+
+perform_unminimization() {
+    local unminimize_cmd="$(command -v unminimize)"
+
+    if [[ "${unminimize_cmd}" != "" ]]; then
+        echo
+        cat <<"END"
+Performing an image unminimization, because we experienced that more and more
+packages have tests depending on data which is not present on a minimized
+image.
+
+If this unminimization bothers you, please remove the
+'perform_unminimization' function from the 'pre-install-deps' hook.
+END
+        echo
```

Comment:
A single echo would do instead of echo+cat+here doc+echo here.

Reply:
Yes it would, but I would not like to change it.